File: python/B4_5300.py
# https://www.acmicpc.net/problem/5300

inp = input()
if inp.isdigit():
    n = int(inp)
    result = ""
    for i in range(1, n+1):
        result += str(i) + " "
        if i % 6 == 0 or i == n:
            result += "Go! "
    print(result)

File: clang/B4_5300.c
//// https://www.acmicpc.net/problem/5300

#include <stdio.h>

int main(void)
{
	int n, i;
	scanf("%d", &n);
	for (i = 1; i <= n; i++)
	{
		printf("%d ", i);
		if (i % 6 == 0 || i == n)
		{
			printf("%s", "Go! ");
		}
	}
	return 0;
}
